meta/runtime.yml

```diff
@@ -1637,6 +1637,13 @@ plugin_routing:
         warning_text: You are using an internal name to access the community.general.keycloak_clientscope
           modules. This has never been supported or documented, and will stop working
           in community.general 9.0.0.
+    identity.keycloak.keycloak_clientsecret:
```

Comment:
Please remove this. Now modules must not add entries here.

Reply:
Done. That was me trying to be courteous to the original PR author, probably the only person who would ever see that deprecation warning.